anago

```diff
@@ -328,6 +328,8 @@ generate_release_notes() {
 
   logecho -n "Checkout master branch to make changes: "
   logrun -s git checkout master || return 1
+  logrun -s git fetch origin master || return 1
```

Comment:
Thanks!

Please add logecho's for these discrete steps as well.

logecho -n "Fetch origin/master: "
and
logecho -n "Rebase origin/master: "